docs/website/docs/developers/general/release-management.md

```diff
@@ -15,6 +15,9 @@ We periodically promote one of these candidates to a "stable" release by
 removing the "pre-release" status. This makes it show up as a "latest" release
 on GitHub. We also push the Python packages for this release to PyPI.
 
+All stable (non-prerelease) releases can be viewed at
+<https://github.com/iree-org/iree/releases?q=prerelease%3Afalse>.
```

Comment:
```suggestion
All stable (non-prerelease) releases can be viewed at GitHub
```suggestion
[releases page](https://github.com/iree-org/iree/releases?q=prerelease%3Afalse).
```
?.
```

Reply:
(fixed the suggestion syntax in your comment)

For this developer page I want to spell out the URL, query parameters and all. In the README I have it hidden under some nicer text:
```markdown
### Release status

Releases notes are
[published on GitHub releases](https://github.com/iree-org/iree/releases?q=prerelease%3Afalse).
```

If we switch this repo to using a "dev-wheels" style release for multiple nightly releases to append assets to then we can switch both back to the base https://github.com/iree-org/iree/releases, since the daily pre-releases won't be spammed